api/data_explorer/__main__.py

```diff
@@ -206,13 +207,31 @@ def _get_es_facets():
             facets[ui_facet_name] = TermsFacet(field=field_name + '.keyword')
         else:
             # Assume numeric type.
-            # TODO: Handle other types.
-            # TODO: Automatically figure out bucket intervals. Unfortunately
-            # Elasticsearch won't do this for us
-            # (https://github.com/elastic/elasticsearch/issues/9572). Make the
-            # ranges easy to read (10-19,20-29 instead of 10-17,18-25).
+            # Creating this facet is a two-step process.
+            # 1) Get max value
+            # 2) Based on max value, determine bucket size. Create
+            #    HistogramFacet with this bucket size.
+            # TODO: When https://github.com/elastic/elasticsearch/issues/31828
+            # is fixed, use AutoHistogramFacet instead. Will no longer need 2
+            # steps.
+            # Step 1: Get max value
+            response = Search(
```

Comment:
Could you write a test for this?

Reply:
If it were easy to test, I would. But it's not easy, and this is throw-away code, so I'd rather not add a test...